src/effector/createEffect.ts

```diff
@@ -133,7 +133,11 @@ export function createEffect<Payload, Done, Fail = Error>(
     return req.req
   }
 
-  const inFlight = (instance.inFlight = createStore(0, {named: 'inFlight'})
+  const inFlight = (instance.inFlight = createStore(0, {
+    named: 'inFlight',
+    // @ts-expect-error
+    serialize: 'ignore',
```

Comment:
I think, that this case is similiar to internal stores of the libraries: if the store is not meant to be transferred over the network, then it is must have `serialize: 'ignore'` set 🤔